### ISSUE TITLE
Quick review - format of triangle gets extended (?)

### ISSUE THREAD

**Issue by ArneSchoch811** [casact/tryangle#2]
Hi there,

I am absolutely new to Python programming so the following issue might have been caused by a lack of Python skills rather than spurious code. Maybe someone else can share his / her insight.

Here is a minimal example for applying the Tryangle package:
```
from sklearn.model_selection import GridSearchCV
from sklearn.pipeline import Pipeline
from tryangle import Development, CapeCod 
from tryangle.metrics import neg_cdr_scorer
from tryangle.model_selection import TriangleSplit
from tryangle.utils.datasets import load_sample

X = load_sample("swiss")

# convert X into a ChainLadder triangle
X_tri = X.triangle

# convert ChainLadder triangle into a dataframe
X_df = X_tri.T

# initialise cross-validation
tscv = TriangleSplit(n_splits=5)

# initalise hyperparameter grids
grid_cc = {
    "dev__n_periods": range(15, 20),
    "dev__drop_high": [True, False],
    "dev__drop_low": [True, False],
    "cc__decay": [0.25, 0.5, 0.75, 0.95]
}

# define pipeline
pipe_cc = Pipeline([("dev", Development()), ("cc", CapeCod())])

# define model
model_cc = GridSearchCV(
    pipe_cc, param_grid=grid_cc, scoring=neg_cdr_scorer, cv=tscv, verbose=1, n_jobs=-1
)

# fit model
fit_cc = model_cc.fit(X, X)

# what was the overall best mean_test_score?
print('Best value Cape Cod:', fit_cc.best_score_)

# now get the predictions by using the optimal CC-settings
fitted_triangle = fit_cc.predict(X).full_triangle_.T
```

The input dataframe has 19 rows and 19 columns. However, the object with the predicted values exhibits 21 rows and 19 columns. Why did it add further rows to the initial dataframe shape?

**Comment by cbalona** [comment]
Hi,

Inspecting both triangles shows that two columns are added at the end of the fitted triangle (note that I have transposed `fitted_triangle` so in my case it has added columns not rows):

```
print(X_tri)
print(fitted_triangle.T)
```

I've omitted the output for brevity, but these last two columns are labelled 240 and 9999. The first, 240, is the LDF applied to the last development year of the swiss triangle (which is 228). So this first column added is simply the expected development after 228. The final added column (9999) is the ultimate claims. `chainladder-python` uses "9999" to signify the ultimate claims column.

So `fitted_triangle` is `X_tri` with the prediction added. In your case, the 2 added rows, are the same as my 2 added columns.
